featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -658,19 +653,22 @@ def _max_hlevel(self, f):
 def check_stacking(primitive, input_types):
```

Comment:
i think we should rename the the `input_types` variable to just `inputs` since it is a list of feature base objects not variable types